src/functions.php

```diff
@@ -35,7 +35,8 @@ function is_zipkin_timestamp($timestamp)
      * @return bool
      */
     function is_zipkin_trace_identifier($identifier) {
-        return ctype_xdigit((string) $identifier) && strlen((string) $identifier) === 32;
```

Comment:
You can modify the TraceSpanIdentifier to allow creating 64bit trace id too, for example by parameter. 
This will be helpful for front-end apps.